libs/wallet-ui/src/contexts/global/global-provider.tsx

```diff
@@ -1,7 +1,7 @@
 import { useMemo } from 'react'
 // @ts-ignore Webpack refuses to build this when imported using a default imoprt
```

Comment:
```suggestion
// @ts-ignore Webpack refuses to build this when imported using a default import
```

(Sorry for such a minor suggestion)

Reply:
no worries, thanks for the catch :)